src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -23,6 +23,7 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.mindrot.jbcrypt.BCrypt;
```

Comment:
I think it's convention to leave an extra line between imports from different packages?

Reply:
got it... thanks!